media-api/app/lib/elasticsearch/SearchFilters.scala

```diff
@@ -82,6 +84,27 @@ trait SearchFilters extends ImageFields {
     "pool"
   )
 
+  val persistedFilter = filters or (
+    filters.bool.must(filters.existsOrMissing("exports", true)),
+    filters.exists(NonEmptyList(identifierField(Config.persistenceIdentifier))),
+    filters.bool.must(filters.boolTerm(editsField("archived"), true)),
+    filters.bool.must(filters.term(usageRightsField("category"), "staff-photographer")),
+    filters.bool.must(filters.term(usageRightsField("category"), "contract-photographer")),
+    filters.bool.must(filters.term(usageRightsField("category"), "commissioned-photographer"))
+  )
+
+  val nonPersistedFilter = filters and (
```

Comment:
Partially - a negation of the `archived` filter isn't strictly correct though as it doesn't cater for the missing case.